dothething/dothething/Assets.xcassets/Placeholder.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "obama.jpg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
The changes in this file add a new image asset named "Placeholder" with three different scale factors (1x, 2x, and 3x). This is a common practice in iOS development to support different device screen resolutions. The asset's filename is set to "obama.jpg", which may need to be updated to a more appropriate name for the placeholder image.